plugin-maven/src/main/java/com/diffplug/spotless/maven/generic/Replace.java

```diff
@@ -0,0 +1,28 @@
+package com.diffplug.spotless.maven.generic;
+
+import com.diffplug.spotless.FormatterStep;
+import com.diffplug.spotless.generic.ReplaceStep;
+import com.diffplug.spotless.maven.FormatterStepConfig;
+import com.diffplug.spotless.maven.FormatterStepFactory;
+import org.apache.maven.plugins.annotations.Parameter;
+
+public class Replace implements FormatterStepFactory {
+
+	@Parameter
```

Comment:
`@Parameter(required = true)` can probably be used for all fields in this class. Then null checks will not be required

Reply:
I used the annotation parameter in my first iteration, but that leads to validation messages which are hard to understand for users:
```
[ERROR] Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:1.11.0-SNAPSHOT:check (default-cli) on project demo: Execution default-cli of goal com.diffplug.spotless:spotless-maven-plugin:1.11.0-SNAPSHOT:check failed: name -> [Help 1]
```

Where as a custom handling is shown like that:
```
[ERROR] Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:1.11.0-SNAPSHOT:check (default-cli) on project demo: Execution default-cli of goal com.diffplug.spotless:spotless-maven-plugin:1.11.0-SNAPSHOT:check failed: Must specify 'name', 'search'
and 'replacement'. -> [Help 1]
```